lm_eval/tasks/hendrycks_test.py

```diff
@@ -85,16 +85,12 @@ def _load_docs(self, filename):
         return (self._convert_standard(doc) for doc in reader)
 
     def training_docs(self):
```

Comment:
We should probably include examples in the auxiliary training folder since those help with fine-tuning a lot.

Reply:
Okay, let's do that 👍 . I wasn't sure if it was worth including since each subject task would have to carry around its own copy of the auxiliary training set.